enderio-base/src/main/java/com/enderio/base/common/recipe/capacitor/CapacitorDataRecipe.java

```diff
@@ -99,20 +101,21 @@ public CapacitorDataRecipe fromJson(@Nonnull ResourceLocation recipeId, JsonObje
         @Nullable
         @Override
         public CapacitorDataRecipe fromNetwork(@Nonnull ResourceLocation recipeId, FriendlyByteBuf buffer) {
+            Item capacitor = ForgeRegistries.ITEMS.getValue(buffer.readResourceLocation());
```

Comment:
fix read/write order. read resourcelocation after reading the cap data, or write resourcelocation first

Reply:
oopsie